spec/models/group_spec.rb

```diff
@@ -128,15 +120,15 @@
       end
 
       it 'should reassign the work package to nobody' do
```

Comment:
The example should be renamed from 
'should reassign the work package to nobody'
to
'should reassign the work package to DeletedUser',
I believe.